client/src/components/AdminContainer/index.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { Layout, Button } from 'antd';
+import logo from '../../assets/images/logo.png';
+import './style.css';
+
+const { Header } = Layout;
+
+const AdminContainer = ({ children, buttonContent, buttonFunction }) => {
+  return (
+    <>
+      <Layout>
+        <Header>
+          <div className="admin-header-image">
+            <img src={logo} alt="logo" />
+          </div>
+          <div className="admin-header-btn">
+            {buttonContent !== undefined && (
```

Comment:
buttonContent has default vlaue so it will never be undefined !!
```suggestion
            {buttonContent && (
```

Reply:
some pages don't have a button in the header so this condition makes the container suitable for all pages if any page doesn't have a button you don't need to add **buttonContent** props.

Reply:
I know what you want to do but this statement below will make the condition always true !!
```
AdminContainer.defaultProps = { buttonContent: 'add cohort', children: 'test' };
```

Reply:
yes you are right, I make it for test and need to modify its default value to undefined or null